### DISCUSSION TITLE
Not able to mint NFT

### DISCUSSION THREAD

**Discussion by Shiv2116** [Cyfrin/foundry-full-course-cu#94] [Q&A]
![ss](https://github.com/Cyfrin/foundry-full-course-f23/assets/114937365/a728dc36-5297-4257-87ae-c9f35a1fbe53)
i am unable to mint nft on sepolia , i am on lesson 9 but unable to mint from the lesson 1

**Comment by alymurtazamemon** [comment]
@Shiv2116 Not sure what happened to this,

@PatrickAlphaC @cromewar can you confirm it?

**Reply by cromewar** [reply]
@Shiv2116 be sure you have enough ETH on the wallet to pay for the gas transaction,

**Comment by Shiv2116** [comment]
i have enough sepolia but still not able to mint nft

**Reply by alymurtazamemon** [reply]
I actually find out, you cannot mint twice with the same address. And where you didn't mint any you should check the challenge information and solve it accordingly.

**Reply by cromewar** [reply]
Nice to know you can do it twice on the same account was this helpful for you @Shiv2116 ?

**Comment by Shiv2116** [comment]
still where is the nft :)

**Reply by alymurtazamemon** [reply]
@Shiv2116 open metamask then nfts tab and import the token, it will take the token contract address and token id.

And arbitrum NFTs you can see on open sea.

**Comment by nik132-eng** [comment]
the same issue facing by me

**Comment by PatrickAlphaC** [comment]
hehehehehehehehehehehehehehehehehehehhehee

You can also come back to it later ;)

**Reply by cromewar** [reply]
👀👀👀 suspicious!

**Comment by irennene** [comment]
Try it later!
